app/invocation/child_invocations.tsx

```diff
@@ -16,13 +16,14 @@ export default class ChildInvocations extends React.Component<ChildInvocationPro
   }
 
   render() {
-    const workflowConfiguredEvent = this.props.model.workflowConfigured;
-    const childInvocationsConfiguredEvent = this.props.model.childInvocationsConfigured;
-    const invocations = childInvocationsConfiguredEvent
-      ? childInvocationsConfiguredEvent.invocation
-      : workflowConfiguredEvent
-      ? workflowConfiguredEvent.invocation
-      : [];
+    const childInvocationConfiguredEvents = this.props.model.childInvocationsConfigured;
+    let invocations = [];
+    for (let i = 0; i < childInvocationConfiguredEvents.length; i++) {
+      const event = childInvocationConfiguredEvents[i];
+      for (let inv of event.invocation) {
+        invocations.push(inv);
+      }
+    }
```

Comment:
should these typescript changes go in a separate PR? at a quick glance they seem unrelated to the `MetadataOnly` support unless I'm missing something?

Reply:
They need to go out before the follow up PR #7138 so I just threw them in here.  But it is a little randomly shoved into here 😅